src/LibraryView.tsx

```diff
@@ -72,4 +75,12 @@ export class LibraryView {
             loadedTypesJson={this.loadedTypesJson}
             layoutSpecsJson={this.layoutSpecsJson} />, htmlElement);
     }
+
+    on(eventName: string, callback: Function) {
+        this.reactor.registerEvent(eventName, callback);
+    }
```

Comment:
Please check with @fanwgwg to see if this `on` method will be mangled (i.e. uglified). If it will be mangled into some other names, then external clients cannot rely on it, get help from Wei Guang to avoid the mangling to be applied on this `on` method. The `raiseEvent` below is internally used so it doesn't matter if it is mangled, leave that as it is.